### PR TITLE
postrelease, codereview, bots docs

### PR DESCRIPTION
**Pull request by loriab**
## Description
management docs

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

File: doc/sphinxman/source/manage_release.rst [hunk 1]
```diff
@@ -80,6 +80,18 @@ Release (e.g., ``v1.3``)
...
+* `Tweak Conda for postrelease`_
+* `Do final pass before release tag`_
+* `Tag postrelease`_
+* `Build Conda Psi4 stack at specific commit`_
+* `Publish to main conda label`_
+* `Build Psi4conda set`_
+* `Generate download page for psicode.org`_
+* `Publish GitHub postrelease`_
+* `Publish psicode release`_
+** `Finalize release`_
```
Comment (robertodr) [inline doc/sphinxman/source/manage_release.rst]: Is this item supposed to be doubly-starred?

Comment (loriab) [reply doc/sphinxman/source/manage_release.rst]: no. I just never got around to doing it for 1.3.1 so the item didn't get polished. :-(

File: doc/sphinxman/source/manage_git.rst [hunk 2]
```diff
@@ -331,3 +334,27 @@ Ideally, each new test or much-altered test should add its own
...
+
+a) Anyone, core-dev or not, is encouraged to review PRs. It's actually good practice for interacting with other open-source projects, where you don't have the advantage of knowing or working with the contributors. Before venturing into projects on GitHub where you don't know the maintainers, it doesn't hurt to read https://snarky.ca/setting-expectations-for-open-source-participation/ .
+
+b) Psi4 is a learning tool for all involved, so partial reviews in areas of confidence and questions and comments on PRs in general are encouraged.
+
+c) Approving before CI completes is fine, though it can be mildly personally embarrasing when CI catches something you didn't.
+
+d) All main branches (master and `1.N.x` maintenance) are protected by GH, including administrators, so even with write access, no one can accidentally push or rewrite the history.
+
+e) GH enforces the 3-revs besides submitter, so fine to merge your own PRs when green.
```
Comment (dgasmith) [inline doc/sphinxman/source/manage_git.rst]: Can we nix a few of these shortcuts? 
```
PR owners who also have maintainer status can merge their PRs as GitHub enforces three external reviewers.
```

File: doc/sphinxman/source/manage_release.rst [hunk 3]
```diff
@@ -265,23 +351,26 @@ Installers are build using the project ``constructor`` and build binary bash scr
...
+* In the end, should have several installers
+
+  ::
 
-  >>> lh build/psi4conda-1.3-py3.*/*64.sh
-  -rwxr-xr-x. 516M Feb 28 20:30 build/psi4conda-1.3-py3.6-linux-64/psi4conda-1.3-py36-Linux-x86_64.sh
-  -rwxr-xr-x. 299M Feb 28 20:31 build/psi4conda-1.3-py3.6-osx-64/psi4conda-1.3-py36-MacOSX-x86_64.sh
-  -rwxr-xr-x. 518M Feb 28 20:30 build/psi4conda-1.3-py3.7-linux-64/psi4conda-1.3-py37-Linux-x86_64.sh
-  -rwxr-xr-x. 299M Feb 28 20:31 build/psi4conda-1.3-py3.7-osx-64/psi4conda-1.3-py37-MacOSX-x86_64.sh
+    >>> lh build/psi4conda-1.3-py3.*/*64.sh
```
Comment (dgasmith) [inline doc/sphinxman/source/manage_release.rst]: `lh` may be a custom shortcut. Isn't standard bash AFAIK.

**Comment by dgasmith** [comment]
This looks awesome! Thanks for work on it.

**Pull request merged**